src/components/calendar/dayCard/DayCardContentView.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+
+const styles = StyleSheet.create({
+  container: {
+    height: 90,
```

Comment:
On my phone with a big screen cards are low. Maybe it would be better to set the percentage height.